modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -7,44 +7,48 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Blacklist
->s|Summary      | Blocks a list or range of IP addresses
+>s|Summary      | Blocks access from a range of IP addresses to an API endpoint
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s|First Mule version available  | 3.8.0
 .1+>.^s| Returned Status Codes
 | 403 - IP is rejected
 |===
 
-The IP blacklist policy can control whether a list or range of IP addresses can access and interact with a configured API endpoint. You use one of the following options to blacklist addresses for your API:
+The IP Blacklist policy controls access to a configured API endpoint from a range of IP addresses. 
 
-* Option 1: Blacklist access based on the IP address of the request
-* Option 2: Blacklist access based on the origin IP address of the x-Forwarded-For header
+The IP Blacklist policy restricts access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
 
-== Option 1: Blacklist access based on the IP address of the request
+Based on your business need, you can specify a DataWeave 2.0 expression for Mule runtime engine 4 (Mule)) or MEL expression for Mule 3.
 
-. Click the instance number of an API.
-+
-. On the API dashboard, click Policies.
-+
-The list of any applied policies appears.
+
+[[specifying_ip_address]]
+== Specifying IP Addresses to Blacklist in the Policy
+
+You can specify the IP addresses to block in the policy using the following ways:
+
+* Blacklist access based on the IP address of the request
+* Blacklist access based on the origin IP address that is determined by resolving a specific DataWeave or MEL expression, such as the https://en.wikipedia.org/wiki/X-Forwarded-For[x-Forwarded-For] header or the `HTTP` header.
```

Comment:
HTTP header is the generic way to mention a header that is part of a HTTP request or response. It is not a special header. A header is composed by a key and a value. The key can be, for example: `x-Forwarded-For`.

Reply:
@nahuel-dallavecchia removed the HTTP part from the sentence. @kbostrom formatting does not work inside []. Done! Thanks!